conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -11,7 +11,18 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
```

Comment:
Should we add `/opt/ghc` to the list below.

Reply:
That directory no longer exists. During testing, I checked the contents of `/opt`. The only directories are
```
2023-07-21T20:10:20.6031001Z + ls /opt
2023-07-21T20:10:20.6050227Z az
2023-07-21T20:10:20.6050577Z containerd
2023-07-21T20:10:20.6050842Z google
2023-07-21T20:10:20.6051086Z hostedtoolcache
2023-07-21T20:10:20.6051337Z microsoft
2023-07-21T20:10:20.6051799Z mssql-tools
2023-07-21T20:10:20.6052069Z pipx
2023-07-21T20:10:20.6052309Z pipx_bin
2023-07-21T20:10:20.6052580Z post-generation
2023-07-21T20:10:20.6052836Z vsts
```

Reply:
These directories can also be candidates for removal!